gi_notifier/models/base.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC, abstractmethod
+from functools import cached_property
+
+from .models import ConstantResource, Drop, MaterialGroup, Weekday
+
+
+class MaterialGroupBase(ABC):
+
```

Comment:
Is there a reason why there is a newline left here while there has not been any in other dataclasses?